bike_shop/data/helpdesk_team.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+  <record id="helpdesk_team_1" model="helpdesk.team">
+    <field name="name">Warranties / After-Sales / SAV</field>
+    <field name="ticket_properties" eval="[{'name': 'order_line_id', 'type': 'many2one', 'domain': '[]', 'string': 'Product', 'comodel': 'sale.order.line', 'default': False, 'view_in_cards': False}, {'name': 'task_id', 'type': 'many2one', 'domain': '[(&quot;is_fsm&quot;, &quot;=&quot;, True)]', 'string': 'Workshop task', 'comodel': 'project.task', 'default': False}]"/>
```

Comment:
Wondering why you have "product" pointing to a SO line, will ask PO about it. But please clean this in the meantime 
```suggestion
    <field name="ticket_properties" eval="[{'name': 'order_line_id', 'type': 'many2one', 'domain': '[]', 'string': 'Product', 'comodel': 'sale.order.line', 'default': False, 'view_in_cards': False}, {'name': 'task_id', 'type': 'many2one', 'domain': '[('is_fsm', '=', True)]', 'string': 'Workshop task', 'comodel': 'project.task', 'default': False}]"/>
```